infrastructure/validators/rewards-payout.md

```diff
@@ -0,0 +1,216 @@
+
+!!!note
+    An uncle block is a Relay Chain block that is valid in every regard, but which failed to become canonical. This can happen when two or more validators are block producers in a single slot, and the block produced by one validator reaches the next block producer before the others. We call the lagging blocks uncle blocks.
+
+Payments occur at the end of every era.
+
+Era points create a probabilistic component for staking rewards.
+
+If the mean of staking rewards is the average rewards per era, then the variance is the variability from the average staking rewards. The exact DOT value of each era point is not known in advance since it depends on the total number of points earned by all validators in a given era. This is designed this way so that the total payout per era depends on Polkadot's [inflation model](), and not on the number of payable actions (for example, authoring a new block) executed. For more information, read this StackExchange post about [reward calculations](https://substrate.stackexchange.com/questions/5353/how-are-rewards-in-dot-calculated-from-the-era-points-earned-by-validators-in-po).
+
+With parachains now on Polkadot, a large percentage of era points will come from parachain validation, as a subset of validators are selected to para-validate for all parachains each epoch, and those para-validators can generate more era points as a result. Para-validators are rewarded 20 era points each for each parachain block that they validate.
+
+In this case, analyzing the expected value of staking rewards will paint a better picture as the weight of era points of validators and para-validators in the reward average are taken into consideration.
+
+!!!note "High-level breakdown of reward variance"
+
+    This should only serve as a high-level overview of the probabilistic nature for staking rewards.
+
+    Let:
+
+    - `pe` = para-validator era points,
+    - `ne` = non-para-validator era points,
+    - `EV` = expected value of staking rewards,
+
+    Then, `EV(pe)` has more influence on the `EV` than `EV(ne)`.
+
+    Since `EV(pe)` has a more weighted probability on the `EV`, the increase in variance against the
+    `EV` becomes apparent between the different validator pools (aka. validators in the active set and
+    the ones chosen to para-validate).
+
+    Also, let:
+
+    - `v` = the variance of staking rewards,
+    - `p` = number of para-validators,
+    - `w` = number validators in the active set,
+    - `e` = era,
+
+    Then, `v` &#8593; if `w` &#8593;, as this reduces `p` : `w`, with respect to `e`.
+
+    Increased `v` is expected, and initially keeping `p` &#8595; using the same para-validator set
+    for all parachains ensures [availability]() and [voting](../learn/learn-polkadot-opengov.md). In addition, despite `v` &#8593; on an `e` to `e` basis, over time, the amount of rewards each validator receives will equal out based on the continuous selection of para-validators.
```

Comment:
IMO it would be better for readability to also explicitly write down these relationships, not just use arrows and icons. However, I do like the arrows, so I'd keep both since I believe they are easier to remember